sqlmesh/core/context.py

```diff
@@ -630,7 +663,15 @@ def render(
 
         model = self.get_model(model_or_snapshot, raise_if_missing=True)
 
-        expand = self.dag.upstream(model.name) if expand is True else expand or []
+        if expand and not isinstance(expand, bool):
+            expand = {
+                normalize_model_name(
```

Comment:
this normalization should not be done here, it should be done in the renderer in resolve tables like it's already done